core/domain/image_validation_services.py

```diff
@@ -88,9 +88,16 @@ def validate_image_and_filename(
                 'The svg tag does not contains the \'xmlns\' attribute.')
     else:
         # Verify that the data is recognized as an image.
-        file_format = imghdr.what(None, h=raw_image)
-        if file_format not in feconf.ACCEPTED_IMAGE_FORMATS_AND_EXTENSIONS:
-            raise utils.ValidationError('Image not recognized')
+        kind = filetype.guess(raw_image)
```

Comment:
Instead of `kind` can we use a variable name that provides more information like `file_format` that was used earlier?